lib/displayHelpers.ts

```diff
@@ -115,43 +115,17 @@ const checkAddress = (input: string, chainAddressPrefix: string) => {
     return error.toString();
   }
 
-  if (prefix !== chainAddressPrefix) {
+  if (!prefix.startsWith(chainAddressPrefix)) {
     return `Expected address prefix '${chainAddressPrefix}' but got '${prefix}'`;
   }
 
-  if (data.length !== 20) {
+  if (data.length < 20) {
```

Comment:
Did you need this change? The validator addresses should all be 20 bytes. This is the data encoded in the bech32 length, not the string length.

Reply:
Probably not then, we did this check that way for tgrade to be on the safe side but it might not be needed

Reply:
There are two possible lengths: 20 an 32 bytes. But 32 bytes is only needed for contracts at the moment.